src/octoprint/static/js/lib/jquery/jquery-ui.js

```diff
@@ -1,6 +1,6 @@
-/*! jQuery UI - v1.12.1 - 2020-11-03
+/*! jQuery UI - v1.12.1 - 2020-11-07
 * http://jqueryui.com
-* Includes: widget.js, widgets/mouse.js
+* Includes: widget.js, position.js, data.js, disable-selection.js, focusable.js, form-reset-mixin.js, jquery-1-7.js, keycode.js, labels.js, scroll-parent.js, tabbable.js, unique-id.js, widgets/draggable.js, widgets/droppable.js, widgets/resizable.js, widgets/selectable.js, widgets/sortable.js, widgets/accordion.js, widgets/autocomplete.js, widgets/button.js, widgets/checkboxradio.js, widgets/controlgroup.js, widgets/datepicker.js, widgets/dialog.js, widgets/menu.js, widgets/mouse.js, widgets/progressbar.js, widgets/selectmenu.js, widgets/slider.js, widgets/spinner.js, widgets/tabs.js, widgets/tooltip.js, effect.js, effects/effect-blind.js, effects/effect-bounce.js, effects/effect-clip.js, effects/effect-drop.js, effects/effect-explode.js, effects/effect-fade.js, effects/effect-fold.js, effects/effect-highlight.js, effects/effect-puff.js, effects/effect-pulsate.js, effects/effect-scale.js, effects/effect-shake.js, effects/effect-size.js, effects/effect-slide.js, effects/effect-transfer.js
```

Comment:
Should this have been checked in? Or is it a way to sneak in #3799? 😜

Reply:
oops...my bad. wasn't paying attention when I committed. Since the CLI is broken, I'll revert that and re-commit and update title to WIP.